packages/system/src/models/library-tag.ts

```diff
@@ -7,6 +7,7 @@ export enum LibraryTag {
 	TESTING = "testing",
 	UTILITIES = "utilities",
 	WEB_SERVERS = "web servers",
+	SERVERLESS = "serverless",
```

Comment:
Shouldn't this be "serverless framework", per the ticket?

Reply:
So do we need this addition here at all then? If it's new from this PR and we're not using it - should it be removed completely?